conf/task_defaults/cowc_counting.yaml

```diff
@@ -0,0 +1,9 @@
+experiment:
+  task: cowc_counting
+  module:
+    model: resnet18
+    learning_rate: 1e-3
```

Comment:
what happens if you start with imagenet weights?

Reply:
Testing now